bosh/README.md

```diff
@@ -6,18 +6,14 @@ If you do not understand a step, please contact human resources.
 
 ## BOSH Introduction
 
-Read the [introduction to BOSH](https://bosh.io/docs#intro).
+Read the [introduction to BOSH](https://bosh.io/docs#intro). [This other resource](https://mariash.github.io/learn-bosh/#introduction) could be helpful too.
 
-## Setup BOSH Lite
+## Setup BOSH-CLI
 
-Go to [https://github.com/cloudfoundry/bosh-deployment](https://github.com/cloudfoundry/bosh-deployment) and install BOSH Lite on your local machine.
+Go to [the BOSH CLI docs](https://bosh.io/docs/cli-v2-install/) and install the BOSH CLI on your local machine.
 
-## Create nginx BOSH Release
+## Fix the given boshrelease
 
-Read the section [Using BOSH to package and distribute software](https://bosh.io/docs/create-release/).
-Then create a BOSH release for nginx. It should display an empty page that is protected by basic authentication.
-Please fork this repository into your GitHub workspace and push your source code to the new repository.
-
-There should be a sample deployment file (e.g. *examples/nginx.yml*) that we can execute to test your BOSH release.
-Please provide a README.md (markdown) in the top directory on how to test your BOSH release with `curl`.
+Read the [documentation for creating releases](https://bosh.io/docs/create-release/) in BOSH. Then try to figure out how to fix the `ruby-app-release` provided under the `./bosh` directory. You should be able to run `create-release --force` and see the release being created.
 
+You should then run the unit-tests from the `./bosh/ruby-app-release/tests` directory,  using `bundle exec rspec job_spec.rb`. Your job is to fix the release so that it conforms to the unit tests, i.e. the tests pass successfully.
```

Comment:
```suggestion
You should then run the unit tests for the job templates. First, you need to install the dependencies using [Bundler](https://bundler.io/) by running `bundle install`. Then, you can execute `bundle exec rspec bosh/ruby-app-release/tests/job_spec.rb`. Your job is to fix the release so that it conforms to the unit tests, i.e. the tests pass successfully.
```

Reply:
The same exact instructions will be provided in a GDoc file. Do we even need the README here?

Reply:
This is a really good question, let's talk about it tomorrow.